_rules/aria-state-or-property-allowed-5c01ea.md

```diff
@@ -5,11 +5,12 @@ rule_type: atomic
 description: |
   This rule checks that WAI-ARIA states or properties are allowed for the element they are specified on.
 accessibility_requirements:
-  wcag20:4.1.2: # Name, Role, Value (A)
+  aria11:state_property_processing:
+    title: ARIA 1.1, 7.6 State and Property Attribute Processing
```

Comment:
Approve that the following from the references WAI-ARIA section sets the requirement for this rule: 

"Global states and properties are supported on any element in the host language. However, authors MUST only use non-global states and properties on elements with a role supporting the state or property; either defined as an explicit WAI-ARIA role, or as defined by the host language implicit WAI-ARIA semantic matching an appropriate WAI-ARIA role."